README.md

```diff
@@ -1,8 +1,63 @@
+
+## Skills
+
+**Proficient:** Java, Linux/Bash, RDBMS/SQL, Agile/Scrum, Git
+
+**Competent:** Python, AWS, C++, C, JavaScript, HTML/CSS, Android, Docker, JSON
+
+**Familiar:** Flask, Google Cloud Platform, Blockchain, Heroku, UML, PHP
+
+## Jarvis Projects
+
+Project source code: [https://github.com/jarviscanada/jarvis_data_eng_OmairFarooqui](https://github.com/jarviscanada/jarvis_data_eng_OmairFarooqui)
+
+
+**Cluster Monitor** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_OmairFarooqui/tree/master/linux_sql)]: Designed an application to record the hardware specifications and monitor resources usages (CPU/memory etc) in real-time of each node within a Linux cluster of 10 nodes. These nodes will be running CentOS 7 and connected via switch, allowing them to communicate through internal IPv4 addresses. The data collected will be stored in a PostgreSQL database and used to generate reports for resource planning. The technologies used in this project include Bash (scripts), IntelliJ IDE, Docker, Git, and PostgreSQL.
+
+**Core Java Apps** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_OmairFarooqui/tree/master/core_java)]:
+      
+  - Grep App: Designed an application which mimics the linux grep command. The purpose of this application is to search through a given directory for any matches to a user-given string and write the found matching strings to a file. The technologies used include: Java 8, IntelliJ IDE, Maven, Docker, and Git.
+  - JDBC App: In progress
+  - Twitter App: Not started
+
+**Springboot App** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_OmairFarooqui/tree/master/springboot)]: Not Started
+
+**Python Data Analytics** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_OmairFarooqui/tree/master/python_data_anlytics)]: Not Started
+
+**Hadoop** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_OmairFarooqui/tree/master/hadoop)]: Not Started
+
+**Spark** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_OmairFarooqui/tree/master/spark)]: Not Started
+
+**Cloud/DevOps** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_OmairFarooqui/tree/master/cloud_devops)]: Not Started
+
+
+## Highlighted Projects
+**IoT Smart Home Security System**: Led a team consisting of four Software Engineering students to develop a Smart Home Security System. Application offers arm/disarm feature, voice command capability via Amazon Alexa, live video camera footage and capability to record footage. Technologies used include Arduino, Java, C, ESP8266 Wi-Fi Module, PIR Motion Sensor, Android, Firebase, and ArduCAM Mini.
```

Comment:
Led a team consisting of four Software Engineering students to develop a Smart Home Security System with/using A, B, C. Application offers arm/disarm feature, voice command capability via Amazon Alexa, live video camera footage and capability to record footage. Technologies used include Arduino, Java, C, ESP8266 Wi-Fi Module, PIR Motion Sensor, Android, Firebase, and ArduCAM Mini.

Algo, meth, agile, test, achievement, results